game/domain/sector.lua

```diff
@@ -138,7 +142,7 @@ function Sector:generate(register, depth)
   -- sector grid generation
   for _,transformer in DB.schemaFor('sector') do
     local spec = self:getSpec(transformer.id)
-    if spec then
+    if spec and transformer.id ~= 'theme' then
```

Comment:
I mean, it's temporary, but ideally it should have been in the `bootstrap` transformer.